model-zoo-api/dict/models.py

```diff
@@ -0,0 +1,117 @@
+modelDict = {"bertsquad-10" : "https://github.com/onnx/models/blob/master/text/machine_comprehension/bert-squad/model/bertsquad-10.onnx?raw=true" ,
```

Comment:
Thank you for this PR. It is an awesome work.
One question:
Are these model urls here hard-coded? 

If yes, I think maybe these urls can be parsed from onnx/models automatically. (finding those onnx models under model/ directory)
In that case, this API can be automatically updated while new models are introduced to model zoo.

Reply:
We used the generate_models.py file to generate the links of the models automatically. It is included in the PR.

Reply:
@kundanapillari - is it necessary to include this dictionary directly? Since we've included the scripts to dynamically generate these files, can they be generated in the initialization step?

The problem is that we'll continually have to keep updating this dictionary of models if we check this in.

Reply:
I think one issue with having the dictionary be generated in the initialization step is that the user has to have an updated cloned repository for it to be able to traverse through the model zoo and create the dict.